lib/DDG/Spice/Crunchbase/Collections.pm

```diff
@@ -0,0 +1,31 @@
+package DDG::Spice::Crunchbase::Collections;
+# Queries the Crunchbase organizations API on crunchbase trigger
+use DDG::Spice;
+
+# Metadata
+primary_example_queries "crunchbase duckduckgo", "crunchbase uber";
+description "Search the Crunchbase database for organizations, people, and products";
+name "Crunchbase";
+source "http://www.crunchbase.com";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Crunchbase/Collections.pm";
+icon_url "https://www.crunchbase.com/favicon.ico";
+category "reference";
+topics "economy_and_finance";
+
+# Attribution 
+attribution github => ["https://github.com/rubinovitz", "rubinovitz"],
+            twitter => ["https://twitter.com/rubinovitz", "rubinovitz"];
+
+# Triggers
+triggers startend => 'crunchbase';
+spice to => 'http://api.crunchbase.com/v/2/organizations?user_key={{ENV{DDG_SPICE_CRUNCHBASE_APIKEY}}}&query=$1';
```

Comment:
I don't have a key yet, but it looks like http://api.crunchbase.com redirects to HTTPS? If it does, please update the `spice to` to "https://api.crunchbase.com/..." to prevent any `301` responses (since we don't handle redirects when we deploy)